### PR TITLE
Updated payjoin-cli/README.md

### PR DESCRIPTION
**Pull request by willowenss**
Created payjoin-cli README with usage instructions based on my experience

### CODE REVIEW

File: payjoin-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+> rpcport=\[rpcport\]\
+> Run the following command from the payjoin-cli directory, where "boom"
+> is the receiving wallet, 18443 is the rpc port, and you wish to
+> request 10,000 sats:\
+> RUST_LOG=debug cargo run \-- -r "http://localhost:18443/wallet/boom"
+> receive 10000\
+> This will generate a pay join-capable bip21 URI with which to accept
+> payjoin as follows:\
+> BITCOIN:BCRT1QCJ4X75DUNY4X5NAWLM3CR8MALM9YAUYWWEWKWL?amount=0.00008888&pj=http
+> s://localhost:3010
```
Comment (DanGould) [inline payjoin-cli/README.md:34]: 8888 != 10000 sats in the request

File: payjoin-cli/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+>
+> Default configuration listens for payjoin requests at
+> http://localhost:3000 and lists server as https://localhost:3010.
+>
+> Download and install local-ssl-proxy:\
```
Comment (DanGould) [inline payjoin-cli/README.md:39]: It'd be nice to have a link to local-ssl-proxy and the reason it is necessary

**Review by DanGould — CHANGES_REQUESTED** [review]
Hell yes! Thanks will

A few points of review

Format using [Github Flavored markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) so that it renders when browsing the repository. Each line here is prefixed with `>` so Github thinks it's all a blockquote I guess.

Config files should be fenced with 

```
```toml
```
blocks and commands with

```
```console
```

blocks

Should be good then.

File: payjoin-cli/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+ The default configuration listens for payjoin requests at http://localhost:3000 and lists the server as 
+ https://localhost:3010. Only https and .onion payjoin endpoints are valid. Therefore, in order to receive
+  payjoin, one must also host an https reverse proxy to marshall https requests from localhost:3010 to
+   localhost:3000.To do this, run:
+```console
+local-ssl-proxy --source 3010 --target 3000
+```
+###  Send Payjoin
+
+ Open a new terminal window and navigate to the  the "sender" directory within payjoin-cli.
```
Comment (DanGould) [inline payjoin-cli/README.md:55]: There is no sender directory in payjoin-cli unless you make one

**Review by DanGould — APPROVED** [review]

**Comment by notmandatory** [comment]
@willowenss remind me to show you how to "squash" commits or use `git commit --amend` so as not to clutter up your PRs commit history. 🙂

**Comment by DanGould** [comment]
> "squash" commits or use git commit --amend so as not to clutter up your PRs commit history. 🙂

When I started getting serious about contributing to rust-bitcoin tcharding introduced me to [this worthwhile read](https://cbea.ms/git-commit/). Mature projects get more anal about commit history for good reason.

**Pull request merged**